exercise2.js

```diff
@@ -0,0 +1,10 @@
+// Write a JavaScript function to check if a given number is prime or not.
+
+
+function isPrime(number){
+    if(number <=1) return false
+    for(let i =2; i<Math.sqrt(number);i++){
+        if(number % i === 0) return false
+    }
+    return true
```

Comment:
It's better to put an empty line before and after a loop if there are some other code after that loop, and always put an empty line before a `return` statement

```suggestion
    if(number <=1) return false
    
    for(let i =2; i<Math.sqrt(number);i++){
        if(number % i === 0) return false
    }
    
    return true
```